src/main/kotlin/com/exactpro/th2/lwdataprovider/handlers/QueueEventsHandler.kt

```diff
@@ -53,7 +54,11 @@ class QueueEventsHandler(
                 handler,
                 batchMaxSize,
             ) {
-                router.sendAll(it, request.externalQueue)
+                if (isBlank(request.externalQueue)) {
```

Comment:
We can not apply you change because the request.externalQueue field is null able.
I see that request has already check null or blank for the externalQueue but we haven't got any grantee that this behavior will not be changed in future. I would like to stay my version for code